versioned_docs/version-1.5.0/release-notes/release-notes-for-kubeslice-oss-1.5.0.mdx

```diff
@@ -0,0 +1,17 @@
+# Release Notes for KubeSlice OSS 1.5.0
+
+*Release date: 19th Apr 2025*
+
+KubeSlice is a ***cloud-independent*** platform that combines network, application, Kubernetes, and deployment services 
+in a framework to accelerate application deployment in a multi-cluster and multi-tenant environment. KubeSlice achieves this 
+by creating logical application ***slice*** boundaries that enable seamless communication between pods and services across 
+clusters, clouds, edges, and data centers.
+
+We continue to add new features and enhancements to KubeSlice. 
+
+## What's New :speaker:
+These release notes describe the new changes and enhancements in this version.
+
+* Support for ARM64 
+* Support for labeling/annotations of user-defined namespaces in KubeSlice Controller helm chart.
```

Comment:
the KubeSlice Controller Helm chart; 

I checked for helm usage, it is always used as `Helm` in title case in running content/paragraphs.